packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -204,21 +204,23 @@ class OpenPositions extends React.Component {
         row_obj.is_unsupported
             ? {
                   component: (
-                      <Localize
-                          i18n_default_text='This trade type is currently not supported on {{website_name}}. Please go to <0>Binary.com</0> for details.'
-                          values={{
-                              website_name,
-                          }}
-                          components={[
-                              <a
-                                  key={0}
-                                  className='link link--orange'
-                                  rel='noopener noreferrer'
-                                  target='_blank'
-                                  href={urlFor('user/portfoliows', undefined, undefined, true)}
-                              />,
-                          ]}
-                      />
+                      <div style={{ position: 'fixed' }}>
```

Comment:
As a general rule, we don't inline styles. Please refactor this to a class reference.